src/app/place/place-list/place-list.page.ts

```diff
@@ -36,11 +36,13 @@ export class PlaceListPage implements OnInit, OnDestroy, AfterViewInit {
   }
 
   private async initialDataLoad() {
-    let loading = await this.loadingCtrl.create({
+    const loading = await this.loadingCtrl.create({
       message: this.translate.instant('loading.loading')
     });
     await loading.present();
-    this.placeService.getPlaces({ limit: this.limit, clearStore: true }).pipe(takeUntil(this.unsubscribe$)).subscribe(async (res: Place[]) => {
+    this.placeService.getPlaces({ limit: this.limit, clearStore: true })
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(async (res: Place[]) => {
       // @hack for hide export item
       setTimeout(async () => {
         this.content.scrollToPoint(0, 48);
```

Comment:
Maybe also add await like in flight-list?

Reply:
done